common/src/main/java/org/opensearch/ml/common/input/Constants.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.input;
+
+public class Constants {
```

Comment:
Seems this class is duplicate of MLConstants. Move to `common` package?

Reply:
It is not duplicate. I separated the constants to two parts. The constant like "train" should be only used by the client. Anyway, I can put them together into the commom.